ExpandedStash/mod.json

```diff
@@ -3,6 +3,79 @@
   "description": "Increases stash size to 16x13. Based on https://www.nexusmods.com/diablo2resurrected/mods/13.",
   "author": "olegbl",
   "website": "https://www.nexusmods.com/diablo2resurrected/mods/173",
-  "version": "1.5"
+  "version": "1.6",
+  "config": [
+    {
+      "id": "IsCustomTabNamesEnabled",
+      "type": "checkbox",
+      "name": "Enable Custom Tab Names",
+      "description": "Set a custom name for each stash tab using the fields below.",
+      "defaultValue": false
+    },
+    {
+      "id": "tabNamePersonal",
+      "type": "text",
+      "name": "Personal Tab",
+      "description": "Set a custom name for the Personal stash tab. Leave blank to use the default name.",
+      "defaultValue": "",
+      "visible": ["eq", ["value", "IsCustomTabNamesEnabled"], true]
```

Comment:
nit: Technically, `["value", "IsCustomTabNamesEnabled"]` already returns a boolean, so you don't actually need the `["eq", true]`, but no harm in it either.